### DISCUSSION TITLE
open-source engine for Heroes of Might and Magic IV

### DISCUSSION THREAD

**Discussion by Brensom** [ihhub/fheroes2#7826] [General]
### Your question

Why did you make the engine open only for this part of the game?
Are you considering doing this for Heroes of Might and Magic IV? This is my favorite part of the series.

**Comment by ihhub** [comment]
@Brensom , it's obvious: these are 2 different games.

**Reply by Brensom** [reply]
@ihhub Yes, But they are from the same series and very similar
